packages/dds/tree/src/shared-tree-core/sharedTreeCore.ts

```diff
@@ -350,28 +347,20 @@ export class SharedTreeCore<TEditor extends ChangeFamilyEditor, TChange> extends
 	}
 
 	protected override reSubmitCore(content: JsonCompatibleReadOnly, localOpMetadata: unknown) {
-		assert(isJsonObject(content), "expected content to be json object");
-		assert(
-			typeof content.revision === "string" && isStableId(content.revision),
-			"expected revision id to be valid stable id",
-		);
-		assert(isStableId(content.revision), "invalid stable ID");
-		const [commit] = this.editManager.findLocalCommit(content.revision);
+		const decoder = (format: number, encodedChange: JsonCompatibleReadOnly) => {
+			return this.changeFamily.encoder.decodeJson(format, encodedChange);
+		};
```

Comment:
For one line closures, you can use the shorter syntax:
```suggestion
		const decoder = (format: number, encodedChange: JsonCompatibleReadOnly) => this.changeFamily.encoder.decodeJson(format, encodedChange);
```

Reply:
same for the below case